doc/src/learn/cryptography/sui-multisig.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Sui Multi-Signature
+---
+
+Sui supports multi-signature (multisig) transactions. A multisig transaction is one that requires more than one private key to authorize it. This topic demonstrate the workflow to create a multisig transaction in Sui, and then submit it using the Sui CLI against a local network. To learn how to set up a local network, see [Sui Local Network](../build/sui-local-network.md).
```

Comment:
Let's add that we support k out of n where n <= 10

Reply:
added